homeassistant/components/climate/radiotherm.py

```diff
@@ -73,6 +73,8 @@ def __init__(self, device, hold_temp):
         self._name = None
         self.hold_temp = hold_temp
         self.update()
+        self._operation_list = ['auto', 'cool', 'heat', 'off']
```

Comment:
Please import these from the climate component: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/climate/__init__.py#L36-L41